frameworks/Crystal/amber/setup.sh

```diff
@@ -6,6 +6,10 @@ shards install
 
 crystal build --release --no-debug src/amber.cr
 
+export GC_MARKERS=1
+
+export GC_NPROCS=1
```

Comment:
> Note that setting this to 1
                when multiple processors are available will preserve
                correctness, but may lead to really horrible performance,
                since the lock implementation will immediately yield without
first spinning.

> It is safer to adjust GC_MARKERS than GC_NPROCS,
since GC_MARKERS has no impact on the lock implementation.